README.md

```diff
@@ -88,34 +106,68 @@ yum install -y epel-release
 yum install -y readline-devel leveldb-devel libffi-devel gcc-c++ redhat-rpm-config gcc python-devel openssl-devel
 ```
 
-### For all of these, make sure that the `Chains` directory in your project has the proper write permissions
+#### Windows
 
-##### Windows
+Help needed. Installing the Python package plyvel seems to require C++ compiler
+support tied to Visual Studio and libraries.
 
-Not sure. Installing the Python package plyvel seems to require C++ compiler support tied to Visual Studio and libraries.
+### Python 3.5
 
--------------------
+neo-python is currently only compatible with **Python 3.5** (due to its `byteplay3` dependency).
 
-### Virtual Environment
+On *nix systems, we can easily install more Python versions on one computer
+using [pyenv](https://github.com/pyenv/pyenv).
 
-Now navigate into the project, make a Python 3 virtual environment and activate
-it via
+After installing `pyenv` (with `brew`, `pip`, or any old package manager), add Python 3.5:
+
+```
+pyenv install 3.5.4
+```
+
+In our `neo-python` directory, make Python 3.5.4 our preferred version:
+
+```
+pyenv local 3.5.4
+```
+
+If we installed `pyenv` properly (especially the `eval "$(pyenv init -)"` part), it
+will automatically switch to Python 3.5.4 whenever we are in the
+neo-python directory.
+
+#### Virtual Environment
+
+It is recommended to put all project dependencies into its own virtual environment,
+this way we don't pollute the global installation which could lead to version conflicts.
 
 ```
 python3.5 -m venv venv
 source venv/bin/activate
 ```
 
-Then install the requirements:
+Now let's install neo-python's dependencies:
 
 ```
-pip install -U setuptools pip wheel
 pip install -e .
 ```
 
+Tip: using [autoenv](https://github.com/kennethreitz/autoenv) to load the configuration
+automatically is a nice little time-saver:
+
+Example `neo-project/.env`:
+```
+eval "$(pyenv init -)"
```

Comment:
Personally I don't want to introduce yet another tool to get things running right now (`pyenv`).